### ISSUE TITLE
Possible stability issues in the batch normalization layer  

### ISSUE THREAD

**Issue by nivshalomlom** [nivshalomlom/ANN-Framework-V3#1]
Check what causes the random instability's (exploding gradient) while using a batch normalization layer.

**Comment by nivshalomlom** [comment]
Found the error, a plus sign instead of minus in the backpropagation code
